### ISSUE TITLE
Displaying a bool message 

### ISSUE THREAD

**Issue by benhirsig** [facontidavide/PlotJuggler#198]
Hello Davide,

I am a new user to PlotJuggler and am using it for plotting some data collected from my robot.

I have a few messages being published whereby the message type is a bool (signifying the power on/off of an electromagnet). I'm trying to plot the change in state against the torque required to run the wheels to see a spike. However, when plotting these bool type messages, I can only plot a line with a gradient, and not a sharp spike similar to that seen in a square wave.

![2019-09-10_13-59-29_01_plot](https://user-images.githubusercontent.com/31052094/64583137-40c93880-d3d3-11e9-8a00-823e53fff7b7.png)

I was wondering if there is a functionality to correctly display the bool as either a constant 1 or 0. Or if there are plans to include this in the works!

Cheers, Ben

**Comment by facontidavide** [comment]
This is expected behavior. If it is easier for you, you can visualize the bool plot as points instead of lines.

For the time being, I am not planning to modify this in any way.

**Comment by benhirsig** [comment]
Thanks for your swift response!

I understand that this won't get changed, but for future users, I came up with a workaround.

In ROS, when I intended to change it to False, I passed the True message followed instantly by the False message, and vice versa. This allowed for the sharp (almost 90 degree) lines slopes on the graph.

![2019-09-10_20-03-31_01_plot](https://user-images.githubusercontent.com/31052094/64604954-78050d00-d406-11e9-9cec-0a97427911fd.png)


Cheers, Ben
